compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -1,8 +1,10 @@
-#![feature(rustc_private)]
 // Note: please avoid adding other feature gates where possible
+#![feature(rustc_private)]
```

Comment:
The comment was intentionally before the feature gate as you did likely add new feature gates after the last one, which is exactly where the comment is positioned.